protocol/src/main/java/org/apache/drill/exec/proto/UserProtos.java

```diff
@@ -47,157 +47,165 @@ public static void registerAllExtensions(
      */
     GOODBYE(2, 2),
     /**
+     * <code>SASL_MESSAGE = 24;</code>
+     *
+     * <pre>
+     * user to bit and bit to user
+     * </pre>
+     */
+    SASL_MESSAGE(3, 24),
```

Comment:
What is the reason we inject SASL_MESSAGE here and we have to update all definitions below?